apps/displays/OSVR_HDK_1_3.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta": {
+    "schemaVersion": 1
+  },
+  "hmd": {
+    "device": {
+      "vendor": "OSVR",
+      "model": "HDK",
+      "num_displays": 1,
+      "Version": "1.2",
+      "Note": "Suitable for HDK 1.0-1.2"
```

Comment:
Never looked at the comments, so they are apt to be aged.  The distortion correction makes the file suitable only for 1.3.